### PR TITLE
General cleaning after removal the ProcessExecutor

### PR DESCRIPTION
**Pull request by avpinchuk**
Remove rudiments after `ProcessExecutor` removal.

### CODE REVIEW

File: appserver/tests/paas/ext-shared-service-enable-disable-test/src/test/java/org/glassfish/tests/paas/enabledisable/ExtSharedServiceEnableDisableTest.java [hunk 1]
```diff
@@ -102,11 +106,12 @@ public void test() throws Exception {
...
                         "--dbhome" , serverContext.getInstallRoot().getAbsolutePath() + File.separator + "databases","--dbhost",ip_address};
-                ProcessExecutor startDatabase = new ProcessExecutor(startdbArgs);
+                ProcessManager startDatabase = new ProcessManager(startdbArgs);
+                startDatabase.setTimeoutMsec(DATABASE_TIMEOUT);
 
                 try {
                     startDatabase.execute();
-                } catch (ExecException e) {
+                } catch (ProcessManagerException e) {
                     e.printStackTrace();
```
Comment (dmatej) [inline appserver/tests/paas/ext-shared-service-enable-disable-test/src/test/java/org/glassfish/tests/paas/enabledisable/ExtSharedServiceEnableDisableTest.java:115]: Wouldn't be better to fail the test? Or we will leave it on another iteration?

File: appserver/tests/paas/ext-shared-service-enable-disable-test/src/test/java/org/glassfish/tests/paas/enabledisable/ExtSharedServiceEnableDisableTest.java [hunk 2]
```diff
@@ -16,37 +17,40 @@
...
+import com.sun.enterprise.universal.process.ProcessManager;
+import com.sun.enterprise.universal.process.ProcessManagerException;
 import com.sun.enterprise.util.OS;
-import com.sun.enterprise.util.ProcessExecutor;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.InputStreamReader;
+import java.io.PrintStream;
+import java.net.*;
```
Comment (dmatej) [inline appserver/tests/paas/ext-shared-service-enable-disable-test/src/test/java/org/glassfish/tests/paas/enabledisable/ExtSharedServiceEnableDisableTest.java:28]: Wildcards are evil :-)

**Review by dmatej — APPROVED** [review]

File: appserver/tests/paas/ext-shared-service-enable-disable-test/src/test/java/org/glassfish/tests/paas/enabledisable/ExtSharedServiceEnableDisableTest.java (hunk 1 above)
Comment (avpinchuk) [reply appserver/tests/paas/ext-shared-service-enable-disable-test/src/test/java/org/glassfish/tests/paas/enabledisable/ExtSharedServiceEnableDisableTest.java:115]: These tests are unused now ;-) I fixes it for consistency. So yes, we leave it refactoring ;-)

File: appserver/tests/paas/ext-shared-service-enable-disable-test/src/test/java/org/glassfish/tests/paas/enabledisable/ExtSharedServiceEnableDisableTest.java (hunk 2 above)
Comment (avpinchuk) [reply appserver/tests/paas/ext-shared-service-enable-disable-test/src/test/java/org/glassfish/tests/paas/enabledisable/ExtSharedServiceEnableDisableTest.java:28]: I now, but these tests inactive now ;-)

**Pull request merged**